dockerfiles/feature-support/hsm-patches/0001-Introduce-generic-keypair-generation-interface-and-e.patch

```diff
@@ -368,7 +369,32 @@ index ec7f279..57cba43 100644
  
  	/* pubkey attributes */
  	pkcs11_addattr(&pubtmpl, CKA_ID, id, id_len);
-@@ -310,6 +320,105 @@ int pkcs11_generate_key(PKCS11_SLOT_private *slot, int algorithm, unsigned int b
+ 	if (label)
+ 		pkcs11_addattr_s(&pubtmpl, CKA_LABEL, label);
+ 	pkcs11_addattr_bool(&pubtmpl, CKA_TOKEN, TRUE);
+-	pkcs11_addattr_bool(&pubtmpl, CKA_ENCRYPT, TRUE);
++	pkcs11_addattr_bool(&pubtmpl, CKA_DERIVE, TRUE);
+ 	pkcs11_addattr_bool(&pubtmpl, CKA_VERIFY, TRUE);
+-	pkcs11_addattr_bool(&pubtmpl, CKA_WRAP, TRUE);
+ 	pkcs11_addattr_var(&pubtmpl, CKA_MODULUS_BITS, num_bits);
+ 	pkcs11_addattr(&pubtmpl, CKA_PUBLIC_EXPONENT, public_exponent, 3);
```

Comment:
public exponent 3 is strongly discouraged. Why do we set this? Do we really need to? The common public exponent for RSA is nowadays 0x10001